great_expectations/render/renderer/column_section_renderer.py

```diff
@@ -33,63 +35,154 @@ def _get_column_name(cls, ge_object):
         if section_name is None:
             column = cls._get_column_name(evrs)
         else:
             column = section_name
 
         content_blocks = []
-        cls._render_header(evrs, column, content_blocks)
-        cls._render_column_type(evrs, content_blocks)
+        cls._render_header(evrs, content_blocks, column_type)
+        # cls._render_column_type(evrs, content_blocks)
         cls._render_overview_table(evrs, content_blocks)
         cls._render_stats_table(evrs, content_blocks)
+
+        cls._render_histogram(evrs, content_blocks)
         cls._render_values_set(evrs, content_blocks)
-        cls._render_unrecognized(evrs, content_blocks)
+        cls._render_bar_chart_table(evrs, content_blocks)
 
-        # FIXME: shown here as an example of bullet list
-        content_blocks.append({
-            "content_block_type": "bullet_list",
-            "bullet_list": [
-                {
-                    "template": "i1",
-                    "params": {}
-                },
-                {
-                    "template": "i2",
-                    "params": {}
-                }
-            ]
-        })
+        # cls._render_statistics(evrs, content_blocks)
+        # cls._render_common_values(evrs, content_blocks)
+        # cls._render_extreme_values(evrs, content_blocks)
+
+        # cls._render_frequency(evrs, content_blocks)
+        # cls._render_composition(evrs, content_blocks)
+
+        cls._render_expectation_types(evrs, content_blocks)
+        # cls._render_unrecognized(evrs, content_blocks)
 
         return {
             "section_name": column,
             "content_blocks": content_blocks
         }
 
     @classmethod
-    def _render_header(cls, evrs, column_name, content_blocks):
+    def _render_header(cls, evrs, content_blocks, column_type=None):
+
+        # NOTE: This logic is brittle
+        try:
+            column_name = evrs[0]["expectation_config"]["kwargs"]["column"]
+        except KeyError:
+            column_name = "Table-level expectations"
+
+        try:
+            column_type_list = cls._find_evr_by_type(
+                evrs, "expect_column_values_to_be_in_type_list"
+            )["expectation_config"]["kwargs"]["type_list"]
+            column_types = ", ".join(column_type_list)
+
+        except TypeError:
+            column_types = "None"
+
+        # assert False
+
         content_blocks.append({
             "content_block_type": "header",
-            "header": column_name,
+            "header": f'{column_name} (type: {column_type})' if column_type else column_name,
+            "sub_header": column_types,
+            # {
+            #     "template": column_type,
+            # },
+            "styling": {
+                "classes": ["col-12"],
+                "header": {
+                    "classes": ["alert", "alert-secondary"]
+                }
+            }
         })
 
+    # @classmethod
```

Comment:
Is this now in header? If so, can we remove this method?

Reply:
Yep